src/cli/argument-parser.js

```diff
@@ -107,6 +107,7 @@ export default class CLIArgumentParser {
             .option('--ports <port1,port2>', 'specify custom port numbers')
             .option('--hostname <name>', 'specify the hostname')
             .option('--proxy <host>', 'specify the host of the proxy server')
+            .option('--proxy-bypass <url1,url2>', 'specify the urls to bypass the proxy server')
```

Comment:
We need to check type of url template value and create error for a case when we cannot parse it

Reply:
I thought it is not required to create error. If rule is incorrent test will run via proxy, and that's all. Rule just will be ignored. Don't you agree?